README.md

```diff
@@ -23,19 +23,44 @@ application to actually see an output of you program (like
 The second option is to wrap the program into `RunCLI` - it will
 emulate terminal output and write it to device display.
 
+## Preparation - Build the Docker Image
+
+For your convenience there is a `docker-compose.yaml`.
+
+```bash
+docker-compose build
+```
+
+will ctreate a `pb-go` container which can be used to compile a go program.
+
+Adjust the `source` path in `docker-compose.yaml` to you needs.
+
+With the current settings you can compile an sqlite test program:
+
+```bash
+docker-compose run --rm pb-go build go-sqlite.go
+```
+
+Alternatively, after building the image, `docker` can be used to compile without adjusting the file.
+
 ## Build an app with UI
 
 To build your app or any example, run (requires Docker):
 
+```bash
+cd ./examples/sqlitetst/
+docker run --rm -v $PWD:/app inkview_pb-go build go-sqlite.go
```

Comment:
I think it's worth to keep the `dennwc/pocketbook-go-sdk` name. This will allow to fetch the image from the Docker Hub instead of building it locally (some people prefer this way).

Also, since Github now has a Docker image storage, it may be worth to move the image there.

Reply:
Changed accordingly.

Regarding the github Docker image storage - I have no clue (yet) about that.

Reply:
Sorry. Have to change a bit more. Will push soon.